sherpa/models/model.py

```diff
@@ -558,7 +564,7 @@ def apply(self, outer, *otherargs, **otherkwargs):
 
 
 class RegriddableModel1D(ArithmeticModel):
-    def regrid(self, *arrays, **kwargs):
+    def regrid(self, *args, **kwargs):
```

Comment:
Given that the `_self` argument is now required, it would be useful to have it visible in the signature, and to have documentation about it.

Reply:
I'm not sure I understand what you are suggesting here.  The only time ```self``` needs to be passed if from the class ```BinaryOpModel``` when it calls the method ```regrid``` of the classes ```RegriddableModel1D``` and ```RegriddableModel2D``` and since the method ```regrid``` can be called by the user, I don't see how one can make ```self``` as required argument.

Reply:
@dtnguyen2 - what I was saying was that I expect there to be some documentation on what the  _self argument is. I mis-spole when I said it was required, what I meant is that it is part of theAPI.